catalogue_of_life/Loader.java

```diff
@@ -140,24 +141,48 @@ private static void giveNamesUniqueID(Path dataDirectory) throws IOException {
 
     private static void separateTaxonDistributions(Path dataDirectory) throws IOException {
```

Comment:
This method is getting a bit long and unwieldy. Can we _separate_ it into meaningful chunks (sub-methods)?

Reply:
Good call